README.md

```diff
@@ -14,9 +14,11 @@ There's another interesting advantage: since this is written in Rust, it might s
 
 ### On-demand block fetching
 
-By connecting to your pruned Bitcoin node through Bitcoin Proxy, your node will now behave as though it is not pruned! If a user or application requires a block that is not retained by your pruned node, Bitcoin Proxy will dynamically fetch the block over the P2P network, then verify its hash against your node to ensure validity.
+By connecting to your pruned Bitcoin node through Bitcoin Proxy, your node will now behave as though it is not pruned. If a user or application requires a block that is not retained by your pruned node, Bitcoin Proxy will dynamically fetch the block over the P2P network, then verify its hash against your node to ensure validity.
 
-The usefulness of this new technology is massive. It means that you can run multiple services against your pruned Bitcoin node — such as Lightning and BTCPay — without them fighting for control over the pruning. Both are happy because both believe they are dealing with an unpruned node.
+This means that you can run multiple services against your _pruned_ Bitcoin node — such as Lightning and BTCPay — without them fighting for control over the pruning. Both are happy because both believe they are dealing with an _unpruned_ node.
```

Comment:
Maybe better "This means that you can run various services that would otherwise require _unpruned_ node against your _pruned_ Bitcoin node — such as Lightning and `electrs`. They are happy because they believe they are dealing with an _unpruned_ node."

Rationale:

* BTCPaySever doesn't need unpruned node anyway
* This has nothing to do with using more than one service
* "that would otherwise require" sounds more understandable to me than "fighting for control"

BTW good point about bandwidth below. :)